tox.ini

```diff
@@ -0,0 +1,42 @@
+[tox]
+isolated_build = True
+envlist =
+    pre-commit
```

Comment:
do these dependencies only get built when running tox then? does it automatically invoke `pip install` on them since we no longer have them in the `pyproject.toml`

Reply:
Yes, installs in this file are only installed by tox when executing the related environment. If a dependency is only needed for a single environment type, I moved them from `pyproject.toml` to here, and I added `tox` to `pyproject.toml`.

Reply:
alright thanks for the clarification